eng/pipelines/extra-platforms/runtime-extra-platforms-ioslikesimulator.yml

```diff
@@ -142,7 +142,7 @@ jobs:
   parameters:
     jobTemplate: /eng/pipelines/common/global-build-job.yml
     helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
-    buildConfig: checked
+    buildConfig: debug
```

Comment:
Why?

Reply:
In Checked configuration, the CORINFO_HELP_TYPEHANDLE_TO_RUNTIMETYPEHANDLE helper target is null, and the stub is not generated. This issue does not occur in Debug configuration. Ideally, we should run it in Checked/Release configuration once we ensure that these optimizations are supported.